lighthouse-core/lib/sentry.js

```diff
@@ -44,7 +50,19 @@ sentryDelegate.init = function init(opts) {
 
     // Special case captureException to return a Promise so we don't process.exit too early
     sentryDelegate.captureException = (...args) => {
-      if (args[0] && args[0].expected) return Promise.resolve();
+      const empty = Promise.resolve();
+      // Ignore expected errors
+      if (args[0] && args[0].expected) return empty;
```

Comment:
feels like it's about time we name the first argument (and maybe second) and spread the remaining.

Reply:
fair